components/drift/common/utils.mjs

```
@@ -0,0 +1,27 @@
+    const isNotEmpyString = typeof value === "string" && value.trim() !== "";
+    const isNotEmptyArray = Array.isArray(value) && value.length;
+    const isNotEmptyObject =
+        typeof value === "object" &&
+        value !== null &&
+        !Array.isArray(value) &&
+        Object.keys(value).length !== 0;
+    isNotEmptyObject && (value = removeNullEntries(value));
+    return ((value || value === false) &&
+        (isNotEmpyString || isNotEmptyArray || isNotEmptyObject || isBoolean || isNumber))
+      ? {
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)